client/lib/abtest/active-tests.js

```diff
@@ -118,5 +118,14 @@ module.exports = {
 		},
 		defaultVariation: 'showFreePlan',
 		allowExistingUsers: false,
+	},
+	firstView: {
+		datestamp: '20160728',
```

Comment:
We need to keep in mind that if we intend to launch this before the 28th, we'll need to update the date here, or else the default variation will be returned until the 28th.